autogen/oai/cohere.py

```diff
@@ -156,7 +157,7 @@ def create(self, params: Dict) -> ChatCompletion:
         cohere_params["preamble"] = preamble
 
         # We use chat model by default
-        client = Cohere(api_key=self.api_key)
+        client = Cohere(api_key=self.api_key, client_name="autogen-cohere")
```

Comment:
can you describe these changes? the client is registered directly from the config_list where intializing.

Reply:
Okay with me, too...

Reply:
Sure, made it configurable with defaults.